lib/jekyll/commands/serve.rb

```diff
@@ -136,7 +136,7 @@ def server_address(server, options = {})
 
         private
         def format_url(ssl_enabled, address, port, baseurl = nil)
-          format("%{prefix}://%{address}:%{port}%{baseurl}", {
+          format("%<prefix>s://%<address>s:%<port>i%<baseurl>s", {
```

Comment:
It's from this cop - https://github.com/bbatsov/rubocop/blob/master/manual/cops_style.md#styleformatstringtoken
by default it enforces annotated tokens (e.g. `%<prefix>s`), but can be confgiured to enforce template tokens (e.g. `%{prefix}`).

Which one do you think we should use?